build-system/travis.js

```diff
@@ -28,99 +28,124 @@ const {red, cyan} = colors;
  * Returns true if this is a Travis build.
  * @return {boolean}
  */
-exports.isTravisBuild = function() {
+function isTravisBuild() {
```

Comment:
I assume you double-checked that these exported functions weren't used anywhere else?